tests/deployed/deployed.spec.ts

```diff
@@ -3,21 +3,19 @@ import path from "path";
 import axios from "axios";
 import { BRIDGE_SERVICE } from "../../src/config";
 import { SignPayload } from "../../src/dtos/secureEnclave";
-import constants from "../../src/@types";
 import * as mockedData from "../data/credentials";
-import { EIDASSignatureOutput } from "../../src/dtos/eidas";
-import { EidasKeysData } from "../../src/dtos/redis";
+import { EidasKeysInput } from "../../src/dtos/redis";
 import { generateDid } from "../utils";
 
 jest.setTimeout(100000);
 
 describe("eidas e2e flow towards vidchain", () => {
   // LOCAL
-  const server = "http://localhost:9002";
+  // const server = "http://localhost:9002";
   // DEV
   // const server = "https://dev.vidchain.net";
   // PRO
-  // const server = "https://api.vidchain.net";
+  const server = "https://api.vidchain.net";
```

Comment:
By default I may leave dev as default so we do not store dummy keys in production.

Reply:
I change it, no prob. but this test is not included on the whole test suite. So, it has to be run manually.